.github/workflows/update-pot-files.yml

```diff
@@ -0,0 +1,38 @@
+
+name: Update source *.pot files from source repo
```

Comment:
Might be worth it to think about whether the name could be made more terse and/or precise. I don't have suggestions, but it'll be copy-pasted and will be appearing in the GHA list, so we just wanna make it nice :slightly_smiling_face:

Reply:
I've tried to clarify the workflow/job names.